lib/codeowners/cli/main.rb

```diff
@@ -37,6 +37,14 @@ def check(repo = '.')
         exit(-1)
       end
 
+      desc 'cleanup REPO', 'Automatically fixes some issues and rewrite .github/CODEOWNERS in a specific order'
```

Comment:
I wonder if we (eventually and not necessary in this PR) want to have the ability to only sort or only remove patterns.  The use-case would be that teams might want to sort it but not removing patterns they don't own, or the other way around. Did you consider that? As I said, no need to prepare that in this PR, just want to make sure we are going in the right direction.

Reply:
It is not that hard. But I'd vote to do that in another PR.
I think it is a good use case. But on the other hand, more options means more flows to support, which leads to harder documentation (we need to explain what each flag does) and greater code complexity. But it is definitely doable and probable has great use cases.

Reply:
I like the idea but I think we're still going into the direction of piping `clean` and `--auto` so, for me makes sense to have this as a real need.